public/manifest.json

```diff
@@ -2,21 +2,21 @@
   "short_name": "React App",
   "name": "Create React App Sample",
   "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
+    // {
+    //   "src": "favicon.ico",
+    //   "sizes": "64x64 32x32 24x24 16x16",
+    //   "type": "image/x-icon"
+    // },
+    // {
+    //   "src": "logo192.png",
+    //   "type": "image/png",
+    //   "sizes": "192x192"
+    // },
+    // {
+    //   "src": "logo512.png",
+    //   "type": "image/png",
+    //   "sizes": "512x512"
+    // }
```

Comment:
```suggestion

```

Reply:
I think we can delete these lines as well as the icons.

Reply:
I uncomment these icons array. It was giving warnings. I'll just leave them as is for now.